easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
```

Comment:
The only reason people pick lowercase module names is to dance around the problem that the most common modules tool has a case-sensitive `module avail`, because it's broken. ;-)

And they're even shooting themselves in the foot there, since I would use `module avail PGI` as a first attempt to see if PGI compilers are available (since `PGI` is how it's spelled all over their website)...

I'd like to hear back from @bartoldeman: do you feel strongly about sticking to `pgi`? If not, I would prefer going with `PGI` instead.